ui/apps/platform/cypress/integration/listeningEndpoints/listeningEndpointsTable.test.js

```diff
@@ -0,0 +1,55 @@
+import withAuth from '../../helpers/basicAuth';
+import { visitListeningEndpointsFromLeftNav } from './ListeningEndpoints.helpers';
+import selectors from './ListeningEndpoints.selectors';
+
+describe('Listening endpoints page table', () => {
+    withAuth();
+
+    it('should correctly display listening endpoint information for central-db', () => {
+        visitListeningEndpointsFromLeftNav();
+
+        // Filter by the 'stackrox' namespace
+        cy.get(selectors.entityDropdownToggle).click();
+        cy.get(`${selectors.entityDropdownMenuItems} button:contains("Namespace")`).click();
```

Comment:
This pair and the following pair, which are repeated for deployment below, are example of trade-off between references to selectors in tests versus possible refactoring as pair of helper functions that encapsulate selectors.